src/test/java/com/commercetools/sync/commons/utils/InventoryEntryCustomUpdateActionUtilsTest.java

```diff
@@ -0,0 +1,50 @@
+package com.commercetools.sync.commons.utils;
+
+import com.commercetools.sync.inventories.InventorySyncOptionsBuilder;
+import com.commercetools.sync.inventories.helpers.InventoryCustomActionBuilder;
+import com.fasterxml.jackson.databind.JsonNode;
+import io.sphere.sdk.client.SphereClient;
+import io.sphere.sdk.commands.UpdateAction;
+import io.sphere.sdk.inventory.InventoryEntry;
+import io.sphere.sdk.inventory.commands.updateactions.SetCustomField;
+import io.sphere.sdk.inventory.commands.updateactions.SetCustomType;
+import org.junit.Test;
+
+import java.util.HashMap;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+
+public class InventoryEntryCustomUpdateActionUtilsTest {
+
+    @Test
+    public void buildTypedSetCustomTypeUpdateAction_WithInventoryResource_ShouldBuildInventoryUpdateAction() {
+        final UpdateAction<InventoryEntry> updateAction =
+            GenericUpdateActionUtils.buildTypedSetCustomTypeUpdateAction("key",
+                new HashMap<>(), mock(InventoryEntry.class), new InventoryCustomActionBuilder(), null,
+                InventoryEntry::getId, inventoryResource -> inventoryResource.toReference().getTypeId(),
+                inventoryResource -> null, InventorySyncOptionsBuilder.of(mock(SphereClient.class)).build())
+                                    .orElse(null);
+
+        assertThat(updateAction).isNotNull();
+        assertThat(updateAction).isInstanceOf(SetCustomType.class);
+    }
+
+    @Test
+    public void buildRemoveCustomTypeAction_WithInventoryResource_ShouldBuildChannelUpdateAction() {
+        final UpdateAction<InventoryEntry> updateAction =
+            new InventoryCustomActionBuilder().buildRemoveCustomTypeAction(null, null);
+
+        assertThat(updateAction).isNotNull();
+        assertThat(updateAction).isInstanceOf(SetCustomType.class);
+    }
+
+    @Test
+    public void buildSetCustomFieldAction_WithInventoryResource_ShouldBuildInventoryUpdateAction() {
+        final UpdateAction<InventoryEntry> updateAction =
+            new InventoryCustomActionBuilder().buildSetCustomFieldAction(null, null, "name", mock(JsonNode.class));
+
+        assertThat(updateAction).isNotNull();
+        assertThat(updateAction).isInstanceOf(SetCustomField.class);
+    }
```

Comment:
check the type of the updateaction here and everywhere ...

Reply:
what do you mean?

Reply:
pls, don't duplicate you question/answer on the similar PR comments. The commenter makes you marks where you should take attention. Even if it is duplication of meaning, it has sense to show you all the places in the code where you should take attention.
But this doesn't mean the requestor has to duplicate the same question and flood other developers with useless notifications.